app/src/components/UserProfile.js

```diff
@@ -64,6 +78,9 @@ export default function UserProfile() {
 
   return (
     <div className="UserProfile">
+      <h3>
+        {state.userName}'s Zone: {state.zone}
```

Comment:
On first load `state.userName` is empty so it looked a bit confusing. Maybe have the default value be the name retrieved from the 0auth login? ( because our gmails have our names ) or just the gmail username part (splice out the @gmail.com) 

Small detail: the `'s` would look better as `’s` ( curly quotes instead of straight quotes )